src/components/sprite-selector/sprite-selector.css

```diff
@@ -68,6 +68,8 @@
     padding-left: calc($space / 2);
     padding-right: calc($space / 2);
     padding-bottom: $space;
+
+    overflow-x: hidden;
```

Comment:
This is needed because on iOS you get a little wiggle of the sprite list when scrolling without it.